tutorials/MD/00_Setup.ipynb

```diff
@@ -0,0 +1,361 @@
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 0.2.2.1 linux/macOS\n",
+    "\n",
+    "On linux/macOS connections to ARCUS can be done directly from the built-in terminal through the `ssh` command:\n",
+    "\n",
+    "```\n",
+    "% ssh username@oscgate.arc.ox.ac.uk \n",
+    "```\n",
+    "\n",
+    "Where `username` is your guest login account username. This will prompt you with a password (the one which was allocated to you alongside your guest account) which you will then enter:\n",
+    "\n",
+    "\n",
+    "![oscgate-password.png](images/setup/ssh/oscgate-password-ssh.png)\n",
+    "\n",
+    "_note: as you can see my username here has been osbcured by a grey bar_\n",
+    "\n",
+    "On first login you may get message with similar wording to:\n",
+    "\n",
+    "`The authenticity of host 'arcus-htc (10.137.128.21)' can't be established. RSA key fingerprint is d1:83:21:b1:f2:bd:???:cd:74:d1:73:???:?a., Are you sure you want to continue connecting (yes/no)?`\n",
+    "\n",
+    "In this case this is perfectly normal, please type `yes` and press \"Enter\". This may also happen when you log into `arcus-htc`.\n",
+    "\n",
+    "This connects you to `oscgate` the gateway computer for accessing ARC resources.\n",
+    "In other words, it's the computer you log into in order to log into other computers owned by ARC.\n",
+    "\n",
+    "\n",
+    "![oscgate-login.png](images/setup/ssh/oscgate-login-ssh.png)\n",
+    "\n",
+    "\n",
+    "On login, you should then follow with:\n",
+    "\n",
+    "```\n",
+    "% ssh arcus-htc\n",
+    "```\n",
+    "\n",
+    "If successfull you should find yourself in your home directory on the ARCUS-HTC computer where we will be running our jobs for the MD tutorial.\n",
+    "\n",
+    "\n",
+    "![arcus-login.png](images/setup/ssh/arcus-htc-login-ssh.png)\n",
+    "\n",
+    "\n",
+    "Typing:\n",
+    "\n",
+    "```\n",
+    "% pwd\n",
+    "```\n",
+    "\n",
+    "Should return: `/home/username` where `username` is your account username.\n",
+    "\n",
+    "This confirms that you are properly connected.\n",
+    "\n",
+    "Next we are going to create a file on ARCUS so that we can then test our ability to copy files back from ARCUS to our own computer.\n",
+    "\n",
+    "To do this we use the echo command:\n",
+    "\n",
+    "```\n",
+    "% echo \"Hello World\" > file.txt\n",
+    "```\n",
+    "\n",
+    "To make sure the file exists you can `more` the file to see its contents\n",
+    "\n",
+    "```\n",
+    "% more file.txt\n",
+    "```\n",
+    "\n",
+    "\n",
+    "![ssh-helloworld.png](images/setup/ssh/ssh-helloworld.png)\n",
+    "\n",
+    "\n",
```

Comment:
I still haven't received a reply regarding a shared directory for the MD input files. Should we just add instructions on how to git clone the repository on ARCUS?
If yes, should the instructions be here you think or in the relevant notebook?

The benefit of having them here would be that they can test that it works well prior to the tutorial. What do you think?

Reply:
@AfroditiMariaZaki 

So two things to tackle here:

1. Do we want to have them clone the whole repository unto ARCUS?

Personally I think we should avoid that, instead I propose we have them clone a new repository that we can set up that only contains the files we want them to use. This way it's clean and we can update it on the fly if there happens to be some big issue (don't put it past any HPC machine to have a last minute change of plans).

2. Should these instructions be in setup?

I think this really should go in the relevant notebook. Whilst it would be good to have them test it in setup, it does mean we have to finalise everything by tomorrow morning. Instead, if we have it being done later on, we have a couple of extra days to respond to any late changes that ARCUS might throw our way.

What do you think?